src/Microsoft.TemplateEngine.Cli/TemplateResolution/TemplateMatchInfoExtensions.cs

```diff
@@ -16,18 +16,13 @@ internal static bool HasDefaultLanguageMatch(this ITemplateMatchInfo templateMat
             return templateMatchInfo.MatchDisposition.Any(x => x.Name == TemplateResolver.DefaultLanguageMatchParameterName && x.Kind == MatchKind.Exact);
         }
 
-        //https://github.com/dotnet/templating/issues/2494
-        //after tab completion is implemented we no longer will be using this match kind - only exact matches will be allowed
-        internal static bool HasAmbiguousParameterValueMatch(this ITemplateMatchInfo templateMatchInfo)
+        internal static bool IsInvokableMatch(this ITemplateMatchInfo templateMatchInfo)
         {
-#pragma warning disable CS0618 // Type or member is obsolete
-            return templateMatchInfo.MatchDisposition.Any(x => x.Kind == MatchKind.AmbiguousValue);
-#pragma warning restore CS0618 // Type or member is obsolete
+            return templateMatchInfo.MatchDisposition.Count > 0
+                            && templateMatchInfo.MatchDisposition.All(x => x.Kind == MatchKind.Exact);
         }
 
-        //https://github.com/dotnet/templating/issues/2494
-        //after tab completion is implemented we no longer will be using this match kind - only exact matches will be allowed
-        internal static bool IsInvokableMatch(this ITemplateMatchInfo templateMatchInfo)
+        internal static bool IsFilterableMatch(this ITemplateMatchInfo templateMatchInfo)
```

Comment:
Can we add documentation on this, explaining what filterable match means? Would be nice to have this also on `IsInvokableMatch`.

Reply:
these methods are dying in refactoring, therefore I would rather ignore it for now as both will be removed in next PR. 
The refactoring brings better naming for these methods and documentation will be included.